crates/cli/tests/snapshots/cli_run__cli_run__platform_switching_rust.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: crates/cli/tests/cli_run.rs
+expression: out.normalize_stdout_and_stderr()
+---
+Roc <3 Rust!🔨 Building host ...
```

Comment:
@lukewilliamboswell should we keep the tests with small simple outputs out of the snapshots? It feels like needless friction in cases like this.